InnerEye/ML/run_ml.py

```diff
@@ -514,11 +512,7 @@ def register_segmentation_model(self,
                                     best_epoch: int,
                                     best_epoch_dice: float,
                                     checkpoint_paths: List[Path],
-                                    model_proc: ModelProcessing,
-                                    run: Optional[Run] = None,
-                                    workspace: Optional[Workspace] = None,
-                                    tags: Optional[Dict[str, str]] = None) -> \
-            Tuple[Optional[Model], Optional[Path], Any]:
+                                    model_proc: ModelProcessing) -> Optional[Tuple[Model, Optional[Any]]]:
```

Comment:
This is inconsistent, ModelDeploymentHookSignature says the hook should return an Optional[Path] not Model.

Reply:
I've simplified the signature. We are not using the deployment hook results anyway.